delivery/flux/README.md

```diff
@@ -277,11 +278,21 @@ xdg-open http://${ADDR}:${PORT}/
 Flux monitors the source git repo and redeploys the application when it detects
 changes.
 
-// TODO: describe how to update to a new version by modifying the git repo
+To update the Flux control plane, run the `flux bootstrap ...` command again; it
+will push a new commit to the repo with the latest component configurations.
+
+To update a HelmRelease managed by Flux, make the desired changes to the chart
+and bump the `version` property in Chart.yaml, then commit these changes. For
+example, change the `replicas` value to `2` in `./deliver/chart/values.yaml`,
+bump the `version` value in `./deliver/chart/Chart.yaml`, commit and push the
```

Comment:
typo : `delivery` instead of `deliver` in both path

Reply:
Fixed!